### PR TITLE
Fix: App wont start because initialy there is not data

### PR DESCRIPTION
**Pull request by DanielVolz**
Quick and dirty fix to create the data directory and download the data, if it doesn't exist. 

Nice dashboard btw!

### CODE REVIEW

File: global.R [hunk 1]
```diff
@@ -10,25 +10,35 @@ library("wbstats")
 
 source("utils.R", local = T)
 
+download <- function() {
```
Comment (chschoenenberger) [inline global.R]: Can we change this to downloadGithubData?
```suggestion
downloadGithubData <- function() {
```

File: global.R [hunk 2]
```diff
@@ -10,25 +10,35 @@ library("wbstats")
...
-    )
-
-    data_path <- "COVID-19-master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-"
-    unzip(
-      zipfile   = "data/covid19_data.zip",
-      files     = paste0(data_path, c("Confirmed.csv", "Deaths.csv", "Recovered.csv")),
-      exdir     = "data",
-      junkpaths = T
-    )
+    download()
```
Comment (chschoenenberger) [inline global.R]:
```suggestion
    downloadGithubData()
```

File: global.R [hunk 3]
```diff
@@ -10,25 +10,35 @@ library("wbstats")
...
+  } 
+  else
+  {
+    dir.create('data')
+    download()
```
Comment (chschoenenberger) [inline global.R]:
```suggestion
    downloadGithubData()
```

**Review by chschoenenberger — CHANGES_REQUESTED** [review]
Never seen this happen but the change is fine with me whith the minor naming change 👍

**Review by chschoenenberger — APPROVED** [review]

**Comment by chschoenenberger** [comment]
Thanks @DanielVolz 
I'll redeploy in a second :)

**Comment by DanielVolz** [comment]
You're welcome!

The problem occurred for me on a win10 machine with the newest R and RStudio (I downloaded it today just to try your awesome app).

**Comment by chschoenenberger** [comment]
I did another minor change. The data was downloaded each time now the app was started, which was a bit annoying while coding. Now it only downloads the data if it is older than 30mins.
If the data folder is not available it is created and downloads the data

**Comment by DanielVolz** [comment]
Yes, this is a much better fix!

**Pull request merged**